components/Form/TelInput.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PhoneInput from 'react-phone-input-2'
+import { Controller } from 'react-hook-form'
+import 'react-phone-input-2/lib/style.css'
+import { Label, Required } from '@/components/Form/common'
+
+const TelInput = ({ control, label, name, required, schema }: any) => {
+  let validPhoneNumber = false
+  const validatePhoneNumber = (
+    inputNumber: string,
+    country: any,
+    isDirty: boolean,
+    phoneLength: number
+  ) => {
+    if (isDirty) {
+      if (
+        inputNumber &&
+        inputNumber?.replace(country.dialCode, '')?.trim() === ''
+      ) {
+        validPhoneNumber = false
+        return false
+      } else if (inputNumber.length < phoneLength) {
+        validPhoneNumber = false
+        return false
+      }
+      validPhoneNumber = true
+      return true
+    }
+    validPhoneNumber = false
+    return false
+  }
+  return (
+    <div className={'pb-5'}>
```

Comment:
strings can be passed directly ```className="pb-5"``` check the whole file for the same thingie please

Reply:
We should also create a chore issue to clean the whole project of classname objects where they're not needed for consistency